tests/zfs-tests/tests/functional/migration/setup.ksh

```diff
@@ -64,10 +45,4 @@ log_must $ZFS set mountpoint=$TESTDIR $TESTPOOL/$TESTFS
 $RM -rf $NONZFS_TESTDIR  || log_unresolved Could not remove $NONZFS_TESTDIR
 $MKDIR -p $NONZFS_TESTDIR || log_unresolved Could not create $NONZFS_TESTDIR
 
-$ECHO "y" | $NEWFS -v ${DEV_DSKDIR}/$NONZFSSIDE_DISK
```

Comment:
For Linux $NEWFS is defined to be `/sbin/mke2fs` so this should work and create a Linux filesystem which can be used in place of UFS for the test.  Maybe the path is wrong here and that's why it's failing.  But there shouldn't be and reason to remove this code and with minimal modification should work for Linux.